kustomization/components/operator-prometheus/subscription.yml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-prometheus
```

Comment:
Never did like the my prefix on names. Can we drop it?

Reply:
Sure; I guess this should really be something like `prometheus-subscription`